Monika After Story/game/script-topics.rpy

```diff
@@ -4412,6 +4411,44 @@ label monika_pets:
     m 1a "Until I find a way out, could you promise to keep me from feeling lonely?"
     m 5a "I'll see if I can get that plush one in here! Oh- don't worry, you're still my favorite~"
     return
+    
+    
+    init 5 python:
+    # List of keywords for the topic.
+    for key in ['dog','dogs','puppy','puppies']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('dog_discussion') # Identifier
+
+label dog_discussion:
+    m 1b "Oh, do you like dogs?"
+    m 1a "Not only are they fun to play with..."
+    m 3i "Having a pet has been shown to help with things like anxiety and depression."
+    m 2j "I actually prefer birds, but I like all animals."
+    m 1o "Though maybe not as much as Natuski did..."
+    m "She was always so embarassed to like cute things. I wish she could have been more accepting of her own interests."
+    m 2q "I suppose her environment had a hand in that."
+    m 1f "If any of your friends have interests they care a lot about, make sure to always be supportive, okay?"
+    m "You never know how much a casual dismissal might hurt someone."
+    m 4e "Knowing you, [player], I'm sure you'd never want to do that."
+    return
+
+
+init 5 python:
+    # List of keywords for the topic.
+    for key in ['cat','cats','kitten','kittens']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('cat_discussion') # Identifier
+
+label cat_discussion:
```

Comment:
We like to keep our label names consistent. All monika topics have a label like `monika_topicname`. Please change your topic's label name so it follows our standards.